auslib/admin/views/rules.py

```diff
@@ -298,14 +299,21 @@ def __init__(self):
 
     @requirelogin
     def _post(self, transaction, changed_by):
-        if request.json and request.json.get("data_version"):
-            form = ScheduledChangeExistingRuleForm()
-        else:
-            form = ScheduledChangeNewRuleForm()
+        if request.json.get("change_type") != "delete":
+            if request.json and request.json.get("change_type") == "update":
+                form = ScheduledChangeExistingRuleForm()
+            elif request.json.get("change_type") == "insert":
+                form = ScheduledChangeNewRuleForm()
```

Comment:
Careful with this block - if change_type is set to something other than "update" or "insert", this will raise an Exception. You should probably return a 400 in an else block here.